.github/workflows/main.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         node-version: '10.x'
     - uses: microsoft/playwright-github-action@v1
     - name: Install dependencies and build
-      run: npm ci
```

Comment:
can we keep ci there?

Reply:
I don't think we should. It is in the way for nowgood reason all the time.